### PR TITLE
Added it-IT translation

### PR DESCRIPTION
**Pull request by roalz**
Gentlemen,

being new to HAML, I hope I've not done something wrong.
Here is the Italian translation for the index.haml, as well as updated CHANGELOG.md and layout/layout.haml.

Hope this helps, keep going with the initiative!

### CODE REVIEW

**Comment by olivierlacan** [comment]
Hey there, do you mind update your PR for the new folder structure? You'll have to move your `it-IT/` directory to the `/0.3.0/` directory and remove your changes to `layout.haml` since that's all dynamic now. Thanks!

**Comment by olivierlacan** [comment]
@roalz Do you have a minute to update this translation, I'd love to release it this week. Thanks. :-)

**Comment by roalz** [comment]
Hi Olivier, I've merged your repo and moved it-IT/index.haml to it-IT/0.30/index.html.haml.
Not sure it's all ok, being new to PRs I feel like I've messed up something, please check.

File: source/it-IT/0.3.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+:markdown
+  # Mantenere un CHANGELOG
+
+  ## Non lasciate che i vostri amici copino e incollino i git log nei CHANGELOG ™
+
+  ### Cos'è un change log?
+  Un change log è un file che contiene una lista curata e ordinata cronologicamente
+  delle modifiche degne di nota per ogni versione di un progetto.
+
+%pre.changelog= File.read(File.expand_path("../../CHANGELOG.md", __FILE__))
```
Comment (olivierlacan) [inline source/it-IT/0.3.0/index.html.haml]: Just change this line to `%pre.changelog= File.read("CHANGELOG.md")`

**Comment by olivierlacan** [comment]
Should be good. Thanks! 👍

**Comment by olivierlacan** [comment]
It's now up on the site, thanks for your contribution @roalz! http://keepachangelog.com/it-IT/0.3.0/

**Pull request merged**